### DISCUSSION TITLE
Change resource limit/request for shared mount pod

### DISCUSSION THREAD

**Discussion by FleetAdmiralButter** [juicedata/juicefs#2882] [Q&A] [answered]
Hi,

I currently have the latest version of the JuiceFS CSI Helm chart installed.

Everything works great with the `storageClassShareMount: true` option (all pods on the same node share a single mount).

However, the resource request/limit of the shared mount seems to inherit what was originally set on the PV.

Is it possible to change these values on the mount shared pod? (or failing that, change the values on the PV itself?)

**Comment by zwwhdls** [comment, marked as answer]
Hi @FleetAdmiralButter , currently, resources of mount pod can only be updated by recreating pvc & sc. Following these steps:

1. delete apps which use juicefs pvc
2. delete pvc & pv
3. update resources in storageClass (refer to [this doc](https://juicefs.com/docs/csi/examples/mount-resources/#dynamic-provisioning))
4. create pvc 
5. create apps

**Reply by FleetAdmiralButter** [reply]
Thanks!
